Clarify how to see your Worker's startup time (#16286)
cloudflare/cloudflare-docs: src/content/docs/workers/platform/limits.mdx

A Worker must be able to be parsed and execute its global scope (top-level code outside of any handlers) within 400 ms. Worker size can impact startup because there is more code to parse and evaluate. Avoiding expensive code in the global scope can keep startup efficient as well.

You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy`, Wrangler will output the startup time of your Worker in the command-line output, using the [`startup_time_ms` field](/api/operations/worker-script-upload-worker-module) returned by the Cloudflare API.

> When you deploy your Worker to Cloudflare using Wrangler, Wrangler returns the startup time

Author: ```suggestion
```

A Worker must be able to be parsed and execute its global scope (top-level code outside of any handlers) within 400 ms. Worker size can impact startup because there is more code to parse and evaluate. Avoiding expensive code in the global scope can keep startup efficient as well.

> You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy`, Wrangler will output the startup time of your Worker in the command-line output, using the [`startup_time_ms` field](/api/operations/worker-script-upload-worker-module) returned by the Cloudflare API.

Author: @danlapid — is this just with `wrangler deploy` or also `wrangler versions upload`?

Reviewer: both, Taylor added the versions one, but with versions you still need the experimental flag, 
<img width="443" alt="image" src="https://github.com/user-attachments/assets/c1808575-7fda-4929-8d2e-67e1fc73111b">

Author: Let's add the note about versions and link to it

> You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy`, Wrangler will output the startup time of your Worker in the command-line output, using the [`startup_time_ms` field](/api/operations/worker-script-upload-worker-module) returned by the Cloudflare API.

Reviewer: ```suggestion
You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy` or `npx wrangler@latest versions upload --experimental-versions`, Wrangler will output the startup time of your Worker in the command-line output, using the `startup_time_ms` field in the [Workers Script API](/api/operations/worker-script-upload-worker-module) or [Workers Versions API](/api/operations/worker-versions-upload-version#request-body).

```

A Worker must be able to be parsed and execute its global scope (top-level code outside of any handlers) within 400 ms. Worker size can impact startup because there is more code to parse and evaluate. Avoiding expensive code in the global scope can keep startup efficient as well.

You can measure your Worker's startup time by deploying it to Cloudflare using [Wrangler](/workers/wrangler/). When you run `npx wrangler@latest deploy`, Wrangler will output the startup time of your Worker in the command-line output, using the [`startup_time_ms` field](/api/operations/worker-script-upload-worker-module) returned by the Cloudflare API.

> 

Author: ```suggestion
```
